src/data/services/EnterpriseDataApiService.js

```diff
@@ -48,6 +48,45 @@ class EnterpriseDataApiService {
       },
     });
   }
+
+  static fetchUnerolledRegisteredLearners(options) {
+    const { enterpriseId } = store.getState().portalConfiguration;
+    const queryParams = {
+      page: 1,
+      page_size: 50,
```

Comment:
nit: should we make a constant for a default page_size?

Reply:
@brittneyexline Potentially, though, I think that could go in it's own PR.